charts/render.py

```diff
@@ -279,10 +279,10 @@ def __call__(self, parser, args, values, option_string=None):
     args = parser.parse_args()
 
     template = subprocess.check_output([
-        'helm', 'template',
-        '--name', args.name,
+        'helm', 'template', args.name,
         '--namespace', args.namespace,
         '--values', args.values,
+        '--include-crds',
         args.path,
     ])
```

Comment:
Should we do an if/else here to check if we are with helm2 or helm3 so that the script is compatible with helm2 as well (or we just expect all charts to be compatible with helm3, I don't know if helm3 has this kind of backward compatibility guaranteed)

Reply:
On my system, both binaries are named `helm`, so I don't see how we can handle both at the same time (except if we ask developers to have both with specific names, or aliases, i.e.: `helm2` and `helm3`).
I think we may need to test if all charts are OK with helm 3, especially since helm 2 is now considered as deprecated.

Reply:
We could easily have both,
But right if all charts are OK with helm3 we should get rid of helm2, I agree